kuma/wiki/templates/wiki/document.html

```diff
@@ -172,7 +172,8 @@
             <a class="button from-search-next only-icon disabled" title="{{ _('Next Search Result') }}" data-empty-title="{{ _('No Previous Search Result') }}"><i aria-hidden="true" class="icon-chevron-right"></i></a>
           </span>
 
-        <h1>{{ document.title }}</h1>
+        {% set safe_title = document.title|forceescape %}
+        <h1>{{ safe_title|addbreaks|safe }}</h1>
```

Comment:
@groovecoder Can you explain to me why this doesn't work?

The `<wbr>` tags are added and not escaped in this code: 
```<h1>{{ "this.is.a.test"|addbreaks|safe }}</h1>```

But in this code the `<wbr>` tags are escaped!?!?
```<h1>{{ safe_title|addbreaks|safe }}</h1>```